src/rard/templates/research/partials/ordered_work_area.html

```diff
@@ -31,10 +33,12 @@
                     <div>
                         {% if work.unknown == True and not flinks and not tlinks and not alinks %}
                         {% elif work.unknown == True %}
-                        <b class='my-3 mr-3'><a href='{% url "work:detail" work.pk %}'>{{ work.name }}</a></b>
+                        <b class='my-3 mr-1'><a href='{% url "work:detail" work.pk %}' >{{ work.name }}</a></b>
```

Comment:
In #381 you suggested it might be nice to have a little badge with the number of links in the collapsed section. I agree.

Reply:
ah yes, thanks for the reminder!

Reply:
<img width="256" alt="image" src="https://github.com/user-attachments/assets/e59d0f7f-eb38-4b4c-9e54-3e9978d730e1" />

do we like it before or after the expand button?

Reply:
<img width="205" alt="image" src="https://github.com/user-attachments/assets/67563bb8-bec0-4c50-b381-16e64ee848ae" />

I suppose the problem with that is the number then gets quite small but I can probably style it

Reply:
![image](https://github.com/user-attachments/assets/41427853-3ddc-47c6-b1df-4bcb68344030)

Reply:
<img width="236" alt="image" src="https://github.com/user-attachments/assets/caa8701f-7d25-4943-a2a6-bb7fb98a4d71" />
<img width="193" alt="image" src="https://github.com/user-attachments/assets/c57b42fe-1ee8-4e13-a0f5-495aa6b62f85" />

I'm liking the outlined version but if you much prefer the filled in then Let me know

Reply:
Oh yeah, outlined is nice